src/config/ethereum/common.ts

```diff
@@ -0,0 +1,43 @@
+import { AppConfig } from 'config/types';
+
+export const commonConfig: AppConfig = {
+  mode: 'development',
+  appUrl: 'https://app.carbondefi.xyz',
+  carbonApi: 'https://api.carbondefi.xyz/v1/',
+  rpcUrl:
+    import.meta.env.VITE_CHAIN_RPC_URL ||
+    'https://eth-mainnet.alchemyapi.io/v2/41hlTcJUXX7kLw5oPejnQJ7V9BdW8Ysg',
+  walletConnectProjectId: 'f9d8863ab6c03f2293d7d56d7c0c0853',
```

Comment:
This id is not currently exposed.

Reply:
It is, just not in Github. Since it's in the client, anyone can access it though their console.